internal/config/config_test.go

```diff
@@ -1037,8 +1037,8 @@ func agentConfig() *Config {
 			},
 		},
 		AllowedDirectories: []string{
-			"/etc/nginx/", "/etc/nginx-agent/", "/usr/local/etc/nginx/", "/var/run/nginx/", "/var/log/nginx/",
-			"/usr/share/nginx/modules/", "/etc/app_protect/",
+			"/etc/nginx/", "/etc/nginx-agent/", "/usr/local/etc/nginx", "/var/run/nginx", "/var/log/nginx",
```

Comment:
Why are we removing the trailing slash on _some_ of the paths in this test?

Reply:
When I played with this briefly it looked like these paths were normlaized and had the trailing slash stripped as config was being parsed and processed. 

This change is confusing though, why remove it from some and not all?